.talismanrc

```diff
@@ -0,0 +1,4 @@
+fileignoreconfig:
+- filename: flake.lock
+  checksum: 9f9722d0a14d372769015f1c57abd64f4bc050ccba75913bc299bab14f6f58fb
+version: ""
```

Comment:
Unless you are also proposing that talisman should be part of the stack (which would require a lot more legwork) can we remove this?

Reply:
removed!